src/utils/service_utils.ts

```diff
@@ -58,7 +58,13 @@ export const serviceUtils = {
         return attributedSwapQuote;
     },
 
-    attributeCallData(data: string, affiliateAddress?: string): string {
+    attributeCallData(
+        data: string,
+        affiliateAddress?: string,
+    ): {
+        affiliatedData: string;
```

Comment:
Yet another naming nit for you :grin: 
I think this should be `affiliateData` (drop the extra `d`).  I realize you just copied it from the variable already in use in the function, but I feel you should correct it there as well, now that the name is being promoted into the interface.

Note that the method giving this value is `affiliateCallDataEncoder.encode()` (no extra `d` there), and you can also note in `git blame` that that variable was added to this existing method, and it seems it was simply typo-ed in that addition.

Reply:
I think the `d` at the end actually is an accurate representation of what it is -- it takes `calldata` and appends affiliate data onto it, so it in my mind it is calldata that has been `affiliated` :)

Reply:
Makes sense to me, thanks for clarifying.